shotgun_api3/shotgun.py

```diff
@@ -833,37 +841,57 @@ def find(self, entity_type, filters, fields=None, order=None,
                                                  include_archived_projects,
                                                  additional_filter_presets)
 
+        if self.server_caps.version and self.server_caps.version >= (3, 3, 0):
```

Comment:
Can we use a `server_caps` `ensure_X` method here (without `raise_hell = True`) to standardize?

Reply:
Sure